cli/options/options.go

```diff
@@ -152,7 +159,11 @@ func GetRPCWithInvoker(gctx context.Context, ctx *cli.Context, signers []transac
 // GetConfigFromContext looks at the path and the mode flags in the given config and
 // returns an appropriate config.
 func GetConfigFromContext(ctx *cli.Context) (config.Config, error) {
-	configPath := "./config"
+	configFile := ctx.String("config-file")
+	if len(configFile) != 0 {
+		return config.LoadFile(configFile)
+	}
+	var configPath = "./config"
```

Comment:
I do love `var` definitions, but I also love consistency. Here you change from `:=` to `var` while introducing another `:=`. It'd be nice to have some similarity.